templates/rhsm-katello-reconfigure.erb

```diff
@@ -98,7 +98,7 @@ else
     sed -i "s/full_refresh_on_yum\s*=.*$/full_refresh_on_yum = 1/g" $CFG
   else
     full_refresh_config="#config for on-premise management\nfull_refresh_on_yum = 1"
-    sed -i "/baseurl/a\n$full_refresh_config" $CFG
+    sed -i "/baseurl/a\ $full_refresh_config" $CFG
```

Comment:
You can also remove the `\`. Otherwise the result is a leading space for the comment:
```console
$ cat test.repo 
[x]
baseurl = bla
sdfsd
$ sed "/baseurl/a\ $full_refresh_config" test.repo 
[x]
baseurl = bla
 #config for on-premise management
full_refresh_on_yum = 1
sdfsd
$ sed "/baseurl/a $full_refresh_config" test.repo 
[x]
baseurl = bla
#config for on-premise management
full_refresh_on_yum = 1
sdfsd
```

Reply:
Done.